contracts/deploy/mainnet/097_native_ssv_staking.js

```diff
@@ -0,0 +1,224 @@
+    const cFeeAccumulator = await ethers.getContractAt(
+      "FeeAccumulator",
+      dFeeAccumulator.address
+    );
+
+    // 6. Init the fee accumulator proxy to point at the implementation, set the governor
+    await withConfirmation(
+      cFeeAccumulatorProxy.connect(sDeployer)[proxyInitFunction](
+        cFeeAccumulator.address, // implementation address
+        addresses.mainnet.Timelock, // governance
+        "0x", // do not call any initialize functions
+        await getTxOpts()
+      )
+    );
+
+    // 7. Safe approve SSV token spending
+    await cStrategy.connect(sDeployer).safeApproveAllTokens();
+
+    // 8. Deploy Harvester
+    const cOETHHarvesterProxy = await ethers.getContract("OETHHarvesterProxy");
+    await deployWithConfirmation("OETHHarvester", [
+      cVaultProxy.address,
+      addresses.mainnet.WETH,
+    ]);
+    const dOETHHarvesterImpl = await ethers.getContract("OETHHarvester");
+
+    console.log("Native Staking SSV Strategy proxy: ", cStrategyProxy.address);
+    console.log(
+      "Native Staking SSV Strategy implementation: ",
+      dStrategyImpl.address
+    );
+    console.log("Fee accumulator proxy: ", cFeeAccumulatorProxy.address);
+    console.log("Fee accumulator implementation: ", cFeeAccumulator.address);
+    console.log(
+      "New OETHHarvester implementation: ",
+      dOETHHarvesterImpl.address
+    );
+
+    // Governance Actions
+    // ----------------
+    return {
+      name: "Deploy new OETH Native Staking Strategy\n\nThis is going to become the main strategy to power the reward accrual of OETH by staking ETH into SSV validators.",
+      actions: [
+        // 1. Add new strategy to vault
+        {
+          contract: cVaultAdmin,
+          signature: "approveStrategy(address)",
+          args: [cStrategyProxy.address],
+        },
+        // 2. configure Harvester to support the strategy
+        {
+          contract: cHarvester,
+          signature: "setSupportedStrategy(address,bool)",
+          args: [cStrategyProxy.address, true],
+        },
+        // 3. set harvester to the strategy
+        {
+          contract: cStrategy,
+          signature: "setHarvesterAddress(address)",
+          args: [cHarvesterProxy.address],
+        },
+        // 4. configure the fuse interval
+        {
+          contract: cStrategy,
+          signature: "setFuseInterval(uint256,uint256)",
+          args: [
+            ethers.utils.parseEther("21.6"),
+            ethers.utils.parseEther("25.6"),
+          ],
+        },
+        // 5. set validator registrator
+        {
+          contract: cStrategy,
+          signature: "setRegistrator(address)",
+          args: [addresses.mainnet.validatorRegistrator],
+        },
+        // 6. set staking threshold
+        {
+          contract: cStrategy,
+          signature: "setStakeETHThreshold(uint256)",
+          // TODO: confirm this number makes sense
+          args: [ethers.utils.parseEther("1024")], // 32ETH * 32
```

Comment:
This is roughly 4 million dollars. May want to start with a lower amount since this essentialy max loss per strategy to a badly behaving strategist. 512 ETH?

Reply:
We can start at 512. The Strategist will have to call `resetStakeETHTally` after 16 validators have been registered. Once we get a roll on the migration we could up the `stakeETHThreshold` to something higher using governance and then drop it back down after the migration.

Reply:
actually, its the 5/8 multisig that needs to call `resetStakeETHTally`